cmp/grid/GridModel.js

```diff
@@ -76,6 +76,8 @@ export class GridModel {
     /** @member {GridGroupSortFn} */
     groupSortFn;
     /** @member {boolean} */
+    enableColumnPinning;
+    /** @member {boolean} */
```

Comment:
Thought about this a bit more, and would like to suggest `enableDragToPin` instead (apologies as I realize I suggested this config name first).

While a bit wordier, I think it's more clear that this config limits the dragging operation only. Columns can still be configured in code to be pinned and, even more so, we're expecting to offer this new mobile chooser option to allow pinning the first column. 

I think that `enableColumnPinning` ends up implying that no pinning will be possible if false, which isn't what we're looking to enforce.